nncf/quantization/advanced_parameters.py

```diff
@@ -190,6 +190,11 @@ class AdvancedQuantizationParameters:
     :type disable_channel_alignment: bool
     :param disable_bias_correction: Whether to disable the bias correction.
     :type disable_bias_correction: bool
+    :param batchwise_statistics: Determines whether quantizer statistics should be calculated
+        for each item of the batch or for the entire batch, default is None.
+    "None" means that if torch.DataLoader or tensorflow.Dataset was passed as a data source for the calibration
```

Comment:
```suggestion
        "None" means that if torch.DataLoader or tensorflow.Dataset was passed as a data source for the calibration
```

Reply:
Do we need to mention framework specific objects in common docs?
May be better write that batch will try get from `batch_size` or `_batch_size` attributes in calibration dataset?

Reply:
done